ci/rpm_build_success.sh

```diff
@@ -0,0 +1,20 @@
+# /bin/sh
+
+# Script to be run on successful RPM build
+
+mydir="$(cd "$(dirname "${BASH_SOURCE[0]}")" >/dev/null 2>&1 && pwd)"
+ci_envs="$mydir/ci/parse_ci_envs.sh"
+if [ -e ${ci_envs}"" ]; 
+  # shellcheck disable=SC1091
+  source "${ci_envs}"
+fi
```

Comment:
(lint) Expected 'then'. [SC1050]
(lint) Expected "#". Fix any mentioned problems and try again. [SC1072]